db/migrations/postgresql/app/V2.0.0.1__table_contacts.sql

```diff
@@ -0,0 +1,18 @@
+BEGIN;
+
+CREATE TABLE IF NOT EXISTS app.contacts (
+    "id"            UUID PRIMARY KEY DEFAULT gen_random_uuid(),
```

Comment:
This is only supported in PostgreSQL > 13.0. Please document that somewhere

Reply:
Readme will be updated with the minimum required version using badge